cleanlab/count.py

```diff
@@ -1381,6 +1381,6 @@ def _get_confident_thresholds_multilabel(
     for class_num, (label, pred_prob) in enumerate(zip(y_one.T, pred_probs.T)):
         pred_probabilitites = stack_complement(pred_prob)
         confident_thresholds[class_num] = get_confident_thresholds(
-            pred_probs=pred_probabilitites, labels=labels
+            pred_probs=pred_probabilitites, labels=label
```

Comment:
can't believe I missed this, thanks to warnings

Reply:
Nice catch!